extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -309,9 +338,69 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
+  buildCallback() {
+    super.buildCallback();
+
+    if (this.isResponsive()) {
```

Comment:
as pointed out offline, i'd prefer measure in onLayoutMeasure, and do the resize in layoutCallback

@dvoytenko